quickstarts/redisenterprise/dashboards/active_active/active_active.json

```diff
@@ -0,0 +1,242 @@
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT average(`crdt.CrdtIngressBytes`) FROM RedisEnterprise FACET entityName SINCE 30 MINUTES AGO TIMESERIES"
+              }
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
+          },
+          "title": "Bandwidth In",
+          "visualization": {
+            "id": "viz.area"
+          }
+        },
+        {
+          "layout": {
+            "column": 7,
+            "height": 2,
+            "row": 4,
+            "width": 6
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT average(`crdt.CrdtEgressBytes`) FROM RedisEnterprise FACET entityName SINCE 30 MINUTES AGO TIMESERIES"
+              }
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
+          },
+          "title": "Bandwidth Out",
+          "visualization": {
+            "id": "viz.area"
+          }
+        },
+        {
+          "layout": {
+            "column": 1,
+            "height": 2,
+            "row": 6,
+            "width": 6
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT average(`crdt.CrdtIngressBytesDecompressed`) FROM RedisEnterprise FACET entityName SINCE 30 MINUTES AGO TIMESERIES"
+              }
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
+          },
+          "title": "Bandwidth In Decompressed",
+          "visualization": {
+            "id": "viz.area"
+          }
+        },
+        {
+          "layout": {
+            "column": 7,
+            "height": 2,
+            "row": 6,
+            "width": 6
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT average(`crdt.CrdtEgressBytesDecompressed`) FROM RedisEnterprise FACET entityName SINCE 30 MINUTES AGO TIMESERIES"
+              }
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
+          },
+          "title": "Bandwidth Out Decompressed",
+          "visualization": {
+            "id": "viz.area"
+          }
+        }
+      ]
+    }
+  }
+}
```

Comment:
I believe this might be indicating that the `json` has two additional ending braces `}` which makes the file invalid.

Reply:
updated - confirmed all are valid json with jq